### DISCUSSION TITLE
Dịch "text tagging"

### DISCUSSION THREAD

**Discussion by cuongvng** [d2l-ai/d2l-vi#2637] [Terminology]
Tên tổng quát của `named entity recognition`, `part-of-speech tagging`...  Có nên dịch là `gán nhãn văn bản` không mọi người?

cc: @minhduc0711 @cuongvng @lkhphuc @davidnvq @rootonchair

**Comment by quangvnai** [comment]
`tag` ở đây chỉ nhãn của một bộ phận trong một input. `nhãn` - `label` thường là nhãn của cả input đấy. Vậy giữ nguyên là `tag` sẽ hợp lý hơn.

**Reply by lkhphuc** [reply]
`gán thẻ văn bản` hay `đánh dấu văn bản` được không nhỉ?

**Reply by cuongvng** [reply]
em thấy `gán thẻ văn bản` ok.

**Reply by thanhcsf** [reply]
Mình vote `gán thẻ văn bản`. Nhờ em @cuongvng tạo PR cho phần thảo luận này nha.
